src/workloads/execution/MixedMultiDeletesDocByDocWithSecondaryIndexes.yml

```diff
@@ -0,0 +1,40 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+  Deletes a range of documents using the DELETE query exec stage both in isolation and while
```

Comment:
Would it be useful to describe that this is with secondary indexes?

Reply:
And maybe why it matters to test with secondary indexes at all.

Reply:
Done! albeit maybe a bit wordy